src/pybind/rados.py

```diff
@@ -1508,6 +1509,22 @@ def get_locator_key(self):
         return self.locator_key
 
 
+    @requires(('snap', Snap))
+    def set_read(self, snap):
```

Comment:
this would be more consistent with the other snap methods if it took the value needed, i.e. the snap_id, rather than a Snap object